lizmap/modules/lizmap/lib/Request/Proxy.php

```diff
@@ -300,7 +313,12 @@ protected static function curlProxy($url, $options)
         }
         if ($options['method'] === 'post') {
             curl_setopt($ch, CURLOPT_POST, 1);
-            curl_setopt($ch, CURLOPT_POSTFIELDS, $options['body']);
+            if (empty($options['body']) && !empty($postfields)) {
+                curl_setopt($ch, CURLOPT_POSTFIELDS, $postfields);
+            }
+            if (!empty($options['body'])) {
```

Comment:
I would write `else if` here.

Any way, I would not do this change, if we store post fields into `$options['body']`